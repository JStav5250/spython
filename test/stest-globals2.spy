int x = 3

def foo()->int:
    int a = x
    str x = "hello"
    return a

type(foo())

str x= "hello"

type(foo())
