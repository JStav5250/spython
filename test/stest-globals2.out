TypeError: expression of type str cannot be assigned to variable 'a' with explicit type int!!
